doc/examples/edges/plot_self_overlapping_polygon.py

```diff
@@ -0,0 +1,60 @@
+"""
+==================================
+Subdivide self-overlapping polygons
+==================================
+
+This example shows how to divide a self-overlapping polygon into non
+self-overlapping sub polygons. Each sub-polygon can be drawn or analyzed by
+separate.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage.measure import divide_selfoverlapping
+from skimage.draw import polygon
```

Comment:
I like importing submodules in the gallery examples (i.e., `from skimage import draw, measure`), so when the reader sees the function calls, they know that `divide_selfoverlapping` is a function `measure` etc. I think it brings some educational value.

Reply:
Absolutely! I updated the example to import the submodules and show that `divide_selfoverlapping` is inside `measure`.